src/main/scala/edu/ie3/simona/config/ConfigFailFast.scala

```diff
@@ -541,6 +549,50 @@ case object ConfigFailFast extends LazyLogging {
     }
   }
 
+  /** Check the validity of control scheme definitions
+    *
+    * @param control
+    *   Control scheme definitions
+    */
+  private def checkControlSchemes(control: Simona.Control): Unit = {
+    control.transformer.foreach(checkTransformerControl)
+  }
+
+  /** Check the suitability of transformer control group definition.
+    *
+    * One important check cannot be performed at this place, as input data is
+    * not available, yet: Do the measurements belong to a region, that can be
+    * influenced by the transformer?
+    *
+    * @param transformerControlGroup
+    *   Transformer control group definition
+    */
+  private def checkTransformerControl(
+      transformerControlGroup: TransformerControlGroup
+  ): Unit = transformerControlGroup match {
+    case TransformerControlGroup(measurements, transformers, vMax, vMin) =>
+      if (measurements.isEmpty)
+        throw new InvalidConfigParameterException(
+          "A transformer control group cannot have no measurements assigned."
+        )
+      if (transformers.isEmpty)
+        throw new InvalidConfigParameterException(
+          "A transformer control group cannot have no transformers assigned."
+        )
+      if (vMin < 0)
+        throw new InvalidConfigParameterException(
+          "The minimum permissible voltage magnitude of a transformer control group has to be positive."
+        )
+      if (vMax < 0)
+        throw new InvalidConfigParameterException(
+          "The maximum permissible voltage magnitude of a transformer control group has to be positive."
+        )
```

Comment:
To discuss: Should we really allow any permissible voltage level that is positive? IMO, we should limit it to +- 10-20% of the 1.0 p.u. mark. I checked if a sophisticated logic is given in **GridModel.scala** and to my understanding, it does not exist. The second reviewer can double-check.

Reply:
Included lower and upper boundary of +-20%. Tests have also been adapted accordingly.